src/languages/sqf.js

```diff
@@ -1402,14 +1507,15 @@ export default function(hljs) {
     'lockCargo',
     'lockDriver',
     'locked',
+    'lockedCameraTo',
     'lockedCargo',
     'lockedDriver',
     'lockedInventory',
     'lockedTurret',
     'lockIdentity',
     'lockInventory',
     'lockTurret',
-    'lockWP',
+    'lockWp',
```

Comment:
```suggestion
    'lockWP',
```
https://community.bistudio.com/wiki/lockWP